cmd/karmabot/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/kamaln7/karmabot"
+	"github.com/kamaln7/karmabot/database"
+	karmabotui "github.com/kamaln7/karmabot/ui"
+	blankui "github.com/kamaln7/karmabot/ui/providers/blank"
+	webui "github.com/kamaln7/karmabot/ui/providers/web"
+
+	"github.com/aybabtme/log"
+	"github.com/nlopes/slack"
+)
+
+func main() {
+	// startup
+
+	ll := log.KV("version", karmabot.VERSION)
+	ll.Info("starting karmabot")
+
+	// config
+
+	flags := struct {
```

Comment:
I'd really rather these flags were all declared in a `var` block at the top - it's a standard Go thing, e.g. https://github.com/kubernetes/kubernetes/blob/ef0c9f0c5b8efbba948a0be2c98d9d2e32e0b68c/cmd/linkcheck/links.go#L40

It means there's a single standard place to see all the flags.